internal/pkg/cli/dialog/inputs.go

```diff
@@ -166,7 +166,7 @@ func newInputsMap() inputsMap {
 	return inputsMap{data: orderedmap.New()}
 }
 
-// inputsMap - map of all Inputs by Input.Id.
+// inputsMap - map of all Inputs by Input.Value.
```

Comment:
Tento komentar si asi nechcel zmenit, ... odporucam vypnut v IDE nahradzovanie v komentaroch.

Reply:
Omg, to jsem fixoval a zapomněl pushnout. 🙈 Pardon.

Reply:
Jo ne, pushnul jsem. To jsem si myslel že je správně. 😅